src/juliaexepath.ts

```diff
@@ -36,10 +37,12 @@ export async function getJuliaExePath() {
             for (let path of pathsToSearch) {
                 try {
                     var res = await exec(`"${path}" -v`);
-
                     actualJuliaExePath = path;
+                    if (actualJuliaExePath == 'julia') {
+                        actualJuliaExePath = which.sync('julia')
```

Comment:
Could you change this to use ``await``? You'll need to promisfy ``which`` to do that. I think [this](https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original) describes quite well how to promisfy a function like ``which`` so that it can be used with ``await``.